src/main/java/frc/robot/Robot.java

```diff
@@ -99,6 +102,8 @@ public void robotPeriodic() {
     // the Command-based framework to work.
     CommandScheduler.getInstance().run();
 
+    m_field.setRobotPose(m_robotContainer.getDriveSubsystem().getPose());
```

Comment:
if we move the field into robot container, we can pass the field as an argument to the Drive and do this update in the drive's periodic